x/mongo/driver/mongocrypt/mongocrypt.go

```diff
@@ -5,7 +5,6 @@
 // a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 
 //go:build cse
-// +build cse
```

Comment:
I think we'll want both of these conditional build tags to be compatible with both versions of tag format.